recipe/conda_build_config.yaml

```diff
@@ -1,70 +1,28 @@
 arm_variant_type: # [aarch64]
   - sbsa          # [aarch64]
 
-# The added keys below are needed to enable ppc64le/aarch64 CUDA builds; the linux64
-# and win64 builds are already handled in the conda-forge-pinning feedstock. Once we
-# migrate to enable CUDA builds on all supported platforms, these can be removed too.
+# conda-forge only supports cross-compiling with CUDA 11.2
```

Comment:
Wonder if we can drop this file entirely after making this change

IIUC this file was used for old CUDA versions on `aarch64` and `ppc64le`, which are no longer relevant after this change

Reply:
Yup. Even for CUDA 11 it's irrelevant after https://github.com/conda-forge/conda-forge-pinning-feedstock/pull/3624

Reply:
Ok dropped in the commit below and re-rendering

Let's see how this goes